application/controllers/UserManagementController.php

```
@@ -1666,10 +1676,10 @@ private function createAdminUser(array $aUser, bool $sendEmail = true): array
      * @todo : move to private function
      *
      * @param array $aUser array with user details
-     * @return array returns all attributes from model user as an array
+     * @return array|null returns all attributes from model user as an array, or null when rendering an error response directly
      * @throws CException
      */
-    public function createNewUser(array $aUser): array
+    public function createNewUser(array $aUser): ?array
```

PHPStan (2.2.7)

[error] PHPDoc tag `@throws` with type CException is not subtype of Throwable


(throws.notThrowable)